### ISSUE TITLE
dialogAppearance.cornerRadius not actually supported

### ISSUE THREAD

**Issue by kjantzer** [Orderella/PopupDialog#175]
The documentation says we can set `PopupDialogDefaultView.appearance().cornerRadius` but Xcode is telling me this is not supported.

**I'm using the latest pod: `0.6.0`**

![screen shot 2017-10-19 at 10 45 59 am](https://user-images.githubusercontent.com/467487/31785671-cb5c5688-b4ba-11e7-8156-5f55ebaa48df.png)

**Comment by mwfire** [comment]
Hey @kjantzer,

you might have actually caught a mistake in the docs ;)
Can you confirm it works with the `PopupDialogContainerView` rather than `PopupDialogDefaultView`?

``` 
let pcv = PopupDialogContainerView.appearance()
pcv.cornerRadius = 10
```

Thanks!

**Comment by kjantzer** [comment]
Yes, `cornerRadius` on PopupDialogContainerView works. (I read that in the docs shortly after submitting this issue)

**Comment by mwfire** [comment]
Released with V0.6.1
